include/deal.II/lac/vector.templates.h

```diff
@@ -1139,6 +1141,18 @@ Vector<Number>::deallocate()
   val = 0;
 }
 
+
+/**
+ * Type trait for a serial vector, i.e. a vector class which is not distributed
+ * over processes.
+ *
+ * @author Uwe Köcher, 2017
+ */
+// template <typename Number>
+// struct is_serial_vector< Vector<Number> > : std_cxx11::true_type
+// {
+// };
+
```

Comment:
yes and the last state is not compiling, but I need the repo for data exchange (my notebook is a bit to slow for compiling deal.II)